artiq/gateware/rtio/phy/ttl_serdes_generic.py

```diff
@@ -7,7 +7,7 @@
 def _mk_edges(w, direction):
     l = [(1 << i) - 1 for i in range(w)]
     if direction == "rising":
-        l = [2**w - 1 ^ x for x in l]
+        l = [((1 << w) - 1) ^ x for x in l]
```

Comment:
This doesn't change anything, right? It should be a separate commit (if it needs to be committed at all).

Reply:
It makes it consistent with the expression two lines above and is a benign clean-up.
Yes. You can ride the style and procedure rulebook on this. But will that improve the code or will it do the opposite?
Does this change bother you in any way? Is it wrong or detrimental?
I don't understand why you are so fussy about style and procedure here.

Reply:
> It makes it consistent with the expression two lines above

Ok then commit it. But I still think it should be separate and not tied with the functional change that detects short pulses.